<ui version="4.0" >
 <class>dialogWPAPersonal</class>
 <widget class="QDialog" name="dialogWPAPersonal" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WPA Personal Config</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="wificonfig.qrc" >
    <normaloff>:/tray_wifi85.png</normaloff>:/tray_wifi85.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>WPA Personal Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Network Key</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineKey" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Network Key (Repeat)</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineKey2" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="checkShowKey" >
        <property name="text" >
         <string>Show Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="spacer12" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>191</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <includes>
  <include location="local" >qmessagebox.h</include>
 </includes>
 <resources>
  <include location="wificonfig.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAPersonal</receiver>
   <slot>slotClose()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkShowKey</sender>
   <signal>clicked()</signal>
   <receiver>dialogWPAPersonal</receiver>
   <slot>slotShowKey()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>129</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
